src/idiomatic/leveraging-the-type-system/borrow-checker-invariants/phantomdata.md

```diff
@@ -0,0 +1,98 @@
+---
+minutes: 10
+---
+
+# Lifetime Relationships primer: PhantomData
+
+Lying to the type system safely, to make things easier for us and encode
+invariants better.
```

Comment:
I don't think we're lying to the type system. We're doing the same thing as we did with ownership and borrow checking: we're using generic parameters on structs in a "non-standard" way. Let's take this angle instead.